packages/config/tsconfig.json

```diff
@@ -1,6 +1,19 @@
 {
   "extends": "../ts/tsconfig.json",
   "include": [
-    "lib/*",
+    "lib",
   ],
+  "compilerOptions": {
+    "outDir": "./dist",
+    "noImplicitAny": true,
+    "strict": true,
+    "allowJs": false,
+    "noUnusedLocals": false,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "noUncheckedIndexedAccess": true,
+    "importsNotUsedAsValues": "error",
+    "declaration": true,
+    "sourceMap": true
+  }
```

Comment:
Some of these like `strict` and `importsNotUsedAsValues` were set in "../ts/tsconfig.json". Do they need re-configured?